expression/builtin_time_test.go

```diff
@@ -1248,3 +1249,27 @@ func (s *testEvaluatorSuite) TestTimeToSec(c *C) {
 		c.Assert(result.GetInt64(), Equals, test.expect)
 	}
 }
+
+func (s *testEvaluatorSuite) TestSecToTime(c *C) {
+	// Test cases from https://dev.mysql.com/doc/refman/5.7/en/date-and-time-functions.html#function_sec-to-time
+	tests := []struct {
+		param  interface{}
+		expect string
+	}{
+		{2378, "00:39:38"},
+		{3864000, "838:59:59"},
+		{-3864000, "-838:59:59"},
+		{86401.4, "24:00:01"},
+		{86401.5, "24:00:02"},
```

Comment:
When I use a invalid string like "abc", the result shows like below.
mysql> select sec_to_time("abc");
+--------------------+
| sec_to_time("abc") |
+--------------------+
| 00:00:00           |
+--------------------+
1 row in set
So do I treat it as a invalid string to return error or just return "00:00:00"?
Still has another question:
How do I add the nil case? I don't know what the expect param should be.
Thank you!